### DISCUSSION TITLE
Is there a way to export profiles and settings?

### DISCUSSION THREAD

**Discussion by swilly97** [FreeTubeApp/FreeTube#5764] [General]
I know I can export subscriptions, history and playlists. I made a separate profile specifically for my music subscriptions, and i unsubscribed from those channels on my normal profile. I was wondering if exporting everything for backups is possible, and if exporting subscriptions/history/playlists while on one profile does it for all profiles.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
See https://docs.freetubeapp.io/usage/data-location/

**Reply by swilly97** [reply]
Thanks! I wasn't even thinking to look for a profiles file. I assumed since it wasn't an option in the data settings to export it, that it was multiple parts.
